git-format-pkg-patch

```diff
@@ -281,35 +281,35 @@ def do_update_patches(args):
                 current_patches[os.path.basename(fname)] = False
                 n_fname = os.path.basename(fname)
                 if not os.path.exists(n_fname):
-                    print "Adding {0} patch".format(fname)
+                    print("Adding {0} patch".format(fname))
                     shutil.copyfile(fname, os.path.join(os.path.abspath("."), n_fname))
                     added.append(n_fname)
                 else:
                     if get_diff_contents(open(fname).read()) != get_diff_contents(open(n_fname).read()):
                         if args.changed:
-                            print "Replacing {0} patch".format(n_fname)
+                            print("Replacing {0} patch".format(n_fname))
                             os.unlink(n_fname)
                             shutil.copyfile(fname, os.path.join(os.path.abspath("."), n_fname))
                             changed.append(n_fname)
                         else:
-                            print "WARNING: Patches {0} and {1} are different!".format(fname, n_fname)
+                            print("WARNING: Patches {0} and {1} are different!".format(fname, n_fname))
 
-    for fname in sorted([patch_name for patch_name, is_dead in current_patches.items() if is_dead]):
-        print "Removing {0} patch".format(fname)
+    for fname in sorted([patch_name for patch_name, is_dead in list(current_patches.items()) if is_dead]):
+        print("Removing {0} patch".format(fname))
         os.unlink(fname)
         removed.append(fname)
 
     # Generate an include for spec changes
     changes = open(CHANGES_FILE, "w")
     for title, data in [('Changed', changed), ('Added', added), ('Removed', removed)]:
         if not data: continue
-        print >> changes, "- {0}:".format(title)
+        print("- {0}:".format(title), file=changes)
```

Comment:
This breaks Python2 compatibility. I would suggest:

```suggestion
        changes.write("- {0}:\n".format(title))
```

Reply:
Why should we stay Python2 compatible?

Reply:
I agree with @brejoc, this is only executed on our dev machines which have Python3.

Reply:
We don't actually need to be Python2 compatible here. Having this Python3 only would work in our particular context.

But, since keeping compatibility here is trivial, then I would consider it.

However, I won't be blocking this PR if we finally make it Python3 only.